### DISCUSSION TITLE
More different music in the game.

### DISCUSSION THREAD

**Discussion by Mirror2013** [sunflower-land/sunflower-land#1162] [Ideas]
Craft boxes with a random music drop (CDs or vinyl records).
Or kraft subscriptions to farmers radio, country, jazz, etc.

So that you can choose the music in the turntable to your liking. Boxes will add excitement. 
When there is the same music from week to week, it starts to get annoying.

An example of how this is implemented in CSGO.
![maxresdefault](https://user-images.githubusercontent.com/34970402/176858375-2d4aebc9-a40e-41f3-9482-80f807a0c69a.jpg)

**Comment by MrSnorch** [comment]
That's a great idea 🚀
